src/api/index.ts

```diff
@@ -271,21 +270,22 @@ router.post(
 		const syncType = req.body.syncType || "partial";
 		// Defaults to anything not completed
 		const statusTypes = req.body.statusTypes as string[] || ["FAILED", "PENDING", "ACTIVE"];
+		// Only trigger a sync for the given installation IDs
+		const installationIds = req.body.installationIds as number[] || [];
 		// Can be limited to a certain amount if needed to not overload system
 		const limit = Number(req.body.limit) || undefined;
 		// Needed for 'pagination'
 		const offset = Number(req.body.offset) || 0;
 
-		// Find all subscriptions that has a status type of X, sorted by last updated
-		const subscriptions: SubscriptionModel[] = await Subscription.findAll({
-			where: {
-				// Does a OR check on status types
-				[Op.or]: statusTypes.map(status => ({ syncStatus: status }))
-			},
-			limit,
-			offset,
-			order: [["updatedAt", "DESC"]]
-		});
+		let subscriptions: SubscriptionModel[];
+
+		if (installationIds.length > 0) {
+			// we want to trigger a sync for the given installation IDs only
+			subscriptions = await Subscription.getAllForInstallations(installationIds, limit, offset);
+		} else {
+			// we want to trigger a sync for all installations with one of the given status types
+			subscriptions = await Subscription.getAllForStatusTypes(statusTypes, limit, offset);
+		}
```

Comment:
Shouldn't it also include the status types? Seems weird that one path will do all installations specified and another is only by status types.   It would make more sense to do both but by default all instances are included.